CODEOWNERS

```diff
@@ -12,4 +12,14 @@
 #  https://help.github.com/en/articles/about-code-owners
 #
 
-*  @bogdandrutu @carlosalberto @SergeyKanzhelev @yurishkuro @tedsuo @iredelmeier @arminru @c24t @reyang @tigrannajaryan @jmacd
+# Global owners, will be the owners for everything in the repo. Membership is tracked via https://github.com/open-telemetry/community/blob/master/community-members.md 
+*   @open-telemetry/specs-approvers
+
+# Trace owners (global + trace)
+opentelemetry/proto/trace/    @open-telemetry/specs-approvers @open-telemetry/specs-trace-approvers
```

Comment:
Can you also add `opentelemetry/proto/collector/trace/v1/` and similarly for other signals?

Reply:
I feel there are very few changes there, they are mostly constant that files.

Reply:
OK, we can add later if needed.